src/plotting/plotSettings/moreOptions/numberInput.component.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import {
+  Unstable_NumberInput as BaseNumberInput,
+  NumberInputProps,
+} from '@mui/base/Unstable_NumberInput';
+import { styled } from '@mui/system';
+import ArrowDropUpIcon from '@mui/icons-material/ArrowDropUp';
+import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown';
+
+const NumberInput = React.forwardRef(function CustomNumberInput(
```

Comment:
Why are we using this custom `NumberInput`? I feel `Textfield` with `type="number"` is sufficient for us, like we use in the shotnumber picker.

Reply:
You're right. I don't know why I created a custom one 😅